README.md

```diff
@@ -1,38 +1,34 @@
+-   [PMEM-CSI for Kubernetes](#pmem-csi-for-kubernetes)
+    -   [About](#about)
+    -   [Design](#design)
+        -   [Architecture and Operation](#architecture-and-operation)
+        -   [DeviceMode:LVM](#devicemodelvm)
+        -   [DeviceMode:Direct](#devicemodedirect)
+        -   [Driver modes](#driver-modes)
+        -   [Driver Components](#driver-components)
+        -   [Communication between components](#communication-between-components)
+        -   [Security](#security)
+        -   [Volume Persistency](#volume-persistency)
+    -   [Prerequisites](#prerequisites)
+        -   [Software required](#software-required)
+        -   [Hardware required](#hardware-required)
+    -   [Supported Kubernetes versions](#supported-kubernetes-versions)
+    -   [Setup](#setup)
+        -   [Get source code](#get-source-code)
+        -   [Build PMEM-CSI](#build-pmem-csi)
+        -   [Run PMEM-CSI on Kubernetes](#run-pmem-csi-on-kubernetes)
+    -   [Automated testing](#automated-testing)
+        -   [Unit testing and code quality](#unit-testing-and-code-quality)
+        -   [QEMU + Kubernetes](#qemu-kubernetes)
```

Comment:
The tool (pandoc?) that was used to generate this version of the TOC is slightly buggy: the correct link here is `#qemu--kubernetes`, not `#qemu--kubernetes`. That wasn't obvious from the diff because each line had changed whitespace.

I'll submit a fix, but long term we'll need to figure out how we want to generate this TOC.